### PR TITLE
Fix Tooltip testkit types

### PR DESCRIPTION
**Pull request by amitdahan**
Fix a mistake where `tooltipTestkitFactory` exported unidriver types instead of legacy ones.

### CODE REVIEW

File: testkit/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import {BaseUniDriver} from 'wix-ui-test-utils/base-driver';
 import {BaseDriver} from 'wix-ui-test-utils/driver-factory';
 import {AvatarDriver} from '../src/Avatar/Avatar.driver';
+import {TooltipUniDriver} from '../src/Tooltip/Tooltip.uni.driver';
```
Comment (mykas) [inline testkit/index.d.ts]: Hey, this should be coming from TooltipNext

Comment (amitdahan) [reply testkit/index.d.ts]: Whoops, correct :(
Any purpose still for `src/Tooltip/Tooltip.uni.driver` then?

Comment (amitdahan) [reply testkit/index.d.ts]: Fixing

Comment (mykas) [reply testkit/index.d.ts]: No purpose for it as we are moving to the next generation Tooltip.

File: testkit/enzyme.d.ts [hunk 2]
```diff
@@ -5,7 +5,8 @@ import {AvatarDriver} from '../src/Avatar/Avatar.driver';
 import {BadgeDriver} from '../src/Badge/Badge.driver';
 import {BadgeSelectDriver} from '../src/BadgeSelect/BadgeSelect.driver';
 import {BoxDriver} from '../src/Box/Box.uni.driver';
-import {TooltipDriver} from '../src/Tooltip/Tooltip.uni.driver';
+import {TooltipUniDriver} from '../src/Tooltip/Tooltip.uni.driver';
```
Comment (mykas) [inline testkit/enzyme.d.ts]: Same is here. Should be coming from TooltipNext folder.

**Comment by amitdahan** [comment]
Fixed, sorry!

**Comment by mykas** [comment]
Looks good. After build is gonna pass - I will push it.

**Comment by amitdahan** [comment]
Thanks @mykas !

**Pull request merged**